Circle.cpp

```diff
@@ -1,14 +1,16 @@
 #ifndef Circle_cpp
 #define Circle_cpp
 #include <iostream>
-#include "Draw.cpp"
+#include "Shape.cpp"
+#include "Area.cpp"
 using namespace std;
-class Circle : public Draw
+class Circle : public Shape
 {
 public:
     void drawShape() override
     {
         cout << "circle" << endl;
+        CaculateCircleArea();
```

Comment:
Why are we calculating area of the circle in draw method?
Draw method should only do one thing: draw.

Reply:
https://github.com/Muhammadali2025222/C-Practice/pull/9/commits/0ed7ccf45db0606bedb3093404d15fc3d9e33a6a
issue resolved
All functions and classes are now performing their designated purpose but their is a slight problem because when i add math.h library in circle.cpp the shapefactory starts to throw an error that i can't instantiate the abstract classes and the error point to new object for shapes the same error occurs even if i use cmath but if i use just math this error doesn't occur but when using just math the PI becomes undeclared